com.unity.perception/Documentation~/FAQ/FAQ.md

```diff
@@ -0,0 +1,584 @@
+        }
+
+        return m_GameObjectBoundsSizeCache.Values.Average();
+    }
+}
+
+  ```
+---
+</details>
+
+<details>
+  <summary><strong>Q: What if I don't want randomized object placement? Can I move my objects in a non-randomized deterministic manner on each frame?</strong> </summary><br>
+
+Even though we call them Randomizers, you can use a Randomizer to perform any task through-out the execution lifecycle of your Scenario. The power of the Randomizers comes from the lifecycle hooks that they have into the Iteration and the Scenario, making it easy to know and guarantee when and in which order in the life of your simulation each piece of code runs. These functions include:
+* `OnEnable`
+* `OnAwake`
+* `OnUpdate`
+* `OnIterationStart`
+* `OnIterationEnd`
+* `OnScenarioStart`
+* `OnScenarioComplete`
+* `OnDisable`
+
+So, in order to have deliberate non-random object movement, you will just need to put your object movement code inside of one of the recurrent lifecycle functions. `OnUpdate()` runs on every frame of the simulation, and `OnIterationStart()` runs every Iteration (which can be the same as each frame if you have only 1 frame per Iteration of your Scenario). For example, the code below moves all objects tagged with the component `ForwardMoverTag` by along their forward axis by 1 unit, on every Iteration. 
+
+```C#
+protected override void OnIterationStart()
+{
+    var tags = tagManager.Query<ForwardMoverTag>();
+    foreach (var tag in tags)
+    {
+        tag.transform.Translate(Vector3.forward);
+    }
+}
+```
+---
+</details>
+
+<details>
+  <summary><strong>Q: The objects instantiated using the sample foreground placement Randomizer are floating in the air. How can I use this Randomizer to place objects on a horizontal surface instead?</strong> </summary><br>
+
+The objects instantiated by the sample foreground Randomizer are all parented to an object named `Foreground Objects` at the root of the Scene Hierarchy. To modify the orientation of the objects, you can simply rotate this parent object at the beginning of the Scenario. 
+
+Alternatively, you could also place `Foreground Objects` inside another GameObject in the Scene using the `Transform.SetParent()` method, and then modify the local position and rotation of `Foreground Objects` in such a way that makes the objects appear on the surface of the parent GameObject. 
+
+To achieve more natural placement, you could also use Unity's physics engine to drop the objects on a surface, let them settle, and then capture an image. To achieve this, you would just need to have sufficient frames in each Iteration of the Scenario (instead of the default 1 frame per iteration), and set your Perception Camera's capture interval to a large enough number that would make it capture each Iteration once after the objects have settled. This example is explained in more detail in the [Perception Camera](#perception-camera) section of this FAQ.
+
```

Comment:
Not that this belongs in the FAQ, but just a thought. We should probably provide a scene (along side the 3box & human pose scenes) that does just this. This probably would be a nice example to include with the package.

Reply:
I agree, this type of setup is tricky to think of without any samples.